### DISCUSSION TITLE
2021/02/27 Community Call

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#994] [Community Call]
1. Leave a new thread below and we will go through the list in order on our community call
2. Feel free to reply to people's threads with thoughts, ideas, support, questions, something you learnt etc. 
3. If you reply in the voice call, please also leave a response on their thread so it is not lost.

Please remember our [Code of Conduct](https://github.com/EddieHubCommunity/support/blob/main/CODE_OF_CONDUCT.md).

*Note: this will be locked until the call*

**Comment by naomi-lgbt** [comment]
If you are a community partner, feel free to give your community a shoutout in a reply to this comment!

**Reply by eddiejaoude** [reply]
Great idea 👍 , thank you ❤️

**Reply by KhushiSharma22** [reply]
Thank You for this! EquiCode is an initiative to cultivate a diverse learning environment and promote beginner centric tech culture! And we love collaborating with EddieHub and other communities!

**Comment by naomi-lgbt** [comment]
Introduce yourself as a reply to this comment:

Tell us about who you are, share your socials, what you are learning, and what you are working on

**Reply by siddharthdayalwal** [reply]
Hey everyone :wave: 

This is Siddharth Dayalwal. Being Passionate for technology and having a desire to learn and explore more makes me what I am.

I love interacting with people and help my community members on a daily basis. Also I am a big hackathon enthusiasts and have participated in 50+ hackathons with different roles! :nerd_face: 

My Socials

- Twitter: <a href="https://twitter.com/siddharth_hacks/">@siddharth_hacks</a>
- Instagram: <a href="https://www.instagram.com/siddharth_hacks/">@siddharth_hacks</a>
- Linkedin: <a href="https://www.linkedin.com/in/siddharth-dayalwal/">in/siddharth-dayalwal/</a>
- Website: <a href="https://siddharth-hacks.live/">siddharth-hacks.live</a> [Hackathon Journey]

**Reply by starlightknown** [reply]
I'm karuna and I'm in first year of my college. I used to learn a lot and when it came to application, my skills lacked . I started attending more and more hackathons which improved my skills and eddiehub helped me really well with open source, I am trying to be consistent. New things I learned are AR (just getting started), django and Blockchain
Twitter - [starlightknown](https://twitter.com/starlightknown?s=09)

**Reply by muthuannamalai12** [reply]
Hello Everyone,
I am Muthu Annamalai, currently pursuing BE in Computer Science and Engineering from PEC Chennai. Being a newbie, I am still learning and therefore I try to use every opportunity to know more about technology. Though I am new, I can vouch for the fact that I am a fast learner with a penchant to both learn and unlearn.
My Socials
- Linkedin: https://www.linkedin.com/in/muthu-annamalai/
- Git Hub: https://github.com/muthuannamalai12
- Twitter: https://twitter.com/muthuannamalai_
- Website: https://muthuannamalai12.github.io/

**Reply by Nitesh-thapliyal** [reply]
Hello everyone,

My name is Nitesh Thapliyal, I am an IT undergrad from India. I am a Tech enthusiast, Learning and exploring new technologies.

- Twitter [@Bauddhik_Geek](https://twitter.com/Bauddhik_Geek) 
- LinkedIn [Nitesh Thapliyal](https://www.linkedin.com/in/nitesh-thapliyal-4403a1135)

Recently started learning Artificial Neural Network, created a simple model to predict the weight of the person using perception, and started learning to use Azure.

**Reply by carrycooldude** [reply]
Hey Everyone, this is Kartikey Rawat, currently pursuing BE in Computer Science and Engineering from BIT Durg. I am Open Source and ML Enthusiast and always ready to collaborate with anyone.
My Socials

-Linkedin: https://www.linkedin.com/in//carrycooldude
-Git Hub: https://github.com/carrycooldude
-Twitter: https://twitter.com/carrycooldude

**Reply by Dhriti-Roy** [reply]
Hello Everyone,
I am Dhriti Roy, currently pursuing B.Tech in Computer Science and  Engineering from Motihari College of Engineering. I'm in second year of my college and I am currently trying to brush my skills more. I started my open-source journey with EddieHub and here we are learning everyday with our community.

- LinkedIn : https://www.linkedin.com/in/dhriti-roy-620a2b19a/
- GitHub : https://github.com/Dhriti-Roy
- Twitter  : https://twitter.com/DhritiRoy19

**Reply by khalidk799** [reply]
Hello Everyone,  I'm Khalid from India, I am pursuing CSE. I've been to development for a while but new to open source. I also Review Tech as a hobby.

Links 

https://www.linkedin.com/in/khalid-khan-4b16aa194

Website- www.thetechinfinite.com

**Comment by naomi-lgbt** [comment]
Reply to *this* comment with any feedback you have for our community calls:

What did you like? What did you not like? What can we do to improve this format?

**Reply by eddiejaoude** [reply]
Great idea 👍

**Comment by khalidk799** [comment]
I've started learning React recently.

**Reply by eddiejaoude** [reply]
Awesome! would you like to share something you found  challenging but then found the solution?

**Reply by naomi-lgbt** [reply]
Where are you learning React from? What are your initial thoughts? Is your first big JS framework?

**Reply by naomi-lgbt** [reply]
I'll always recommend freeCodeCamp 🙂

**Reply by stemount** [reply]
@nhcarrigan I think I'm gonna try freecodecamp after learning about that awesome "I need help" function that will save your code you are hacking together, that sounds ABSOLUTELY amazing!!!!!!

**Comment by KhushiSharma22** [comment]
I do have a question! What's the plan for the next few months for EddieHub? Anything new?

Also, just adding to this, I have been focusing on contributing more towards communities and have been working closely with different initiatives! I have a few exciting ideas for different projects and some activities for EquiCode..

**Reply by eddiejaoude** [reply]
Yay! Let's discuss 🎉 maybe the community has ideas too 🤓

**Reply by naomi-lgbt** [reply]
This is a great question! I'll follow up with:

This is *your* community too - what would you like to see on the roadmap?

**Reply by stemount** [reply]
@eddiejaoude I'm just gonna remind you about that world map overlay project ;) I was enjoying watching the pain of everyone figuring out how to make a nice map overlay. It's hard!

**Comment by oreo-gif** [comment]
I started with Javascript recently at FreeCodeCmp.(I'm a complete noob) Basics were easy but I'm messing up with ES6.

**Reply by naomi-lgbt** [reply]
JavaScript is a lot to learn.

What challenges are you facing? What questions do you have? What steps are you taking to support your learning?

**Reply by adityaraute** [reply]
Did you know that @nhcarrigan works with FreeCodeCamp?

**Reply by oreo-gif** [reply]
That's the second topic in The certification. I mean I'm Messed with constructors destructors promises

**Reply by oreo-gif** [reply]
I'm messed up about why using so many ways to do the same thing. I'm getting confused between their use

**Reply by eddiejaoude** [reply]
Maybe build a smaller project with a single goal? It might be more fun which really helps

**Reply by oreo-gif** [reply]
Okay. I'll start from tomorrow itself

**Reply by eddiejaoude** [reply]
Yay! Awesome! Any questions let us know, someone in the community will be able to help 🤓 - the more specific the question the better

**Reply by oreo-gif** [reply]
sure! thanks a lot!!

**Comment by siddharthdayalwal** [comment]
@eddiejaoude tell us more about EddieCon community event and how can EddieHub help you in that! :smile:

**Reply by siddharthdayalwal** [reply]
I have seen few interesting stuffs here though : https://github.com/EddieHubCommunity/support/discussions/789

**Reply by naomi-lgbt** [reply]
It's currently in the initial planning stages - once we've laid down the foundations, we'll open up some discussions for what folks would like to see.

**Reply by eddiejaoude** [reply]
I think we should have a dedicate call for this to capture peoples ideas 👍 but they can also leave their ideas on #789 👍

**Comment by Rahma-AlWadhahi** [comment]
I don’t want to speak this time, but one recent thing I have learned from [yesterday’s livestream ](https://youtu.be/pekHBfYd1-Q) is using ***suggestion*** key with the code block

**Reply by naomi-lgbt** [reply]
The suggestion feature is *so* helpful - I use it regularly!

**Reply by eddiejaoude** [reply]
thanks for sharing @Rahma-AlWadhahi 👍

**Reply by naomi-lgbt** [reply]
![image](https://user-images.githubusercontent.com/63889819/109396570-2183db00-78e7-11eb-804f-ce8347219e9c.png)

**Reply by stemount** [reply]
@nhcarrigan @Rahma-AlWadhahi this is really great functionality because it's a "kinder" way to suggest a small change over creating a PR on top of someone elses PR (which seems more of an insult than it really is - even if it is correct)!

I also find that people find PRs into PRs confusing in general because you need to sort of be a Git master sometimes with complicated merges of suggestions, and just pasting a block of code without commenting why is criminal use of PR review feedback!

**Comment by siddharthdayalwal** [comment]
With @KhushiSharma22 talking about collaboration, one thing came up in mind. As I am quite active with hackathon communities, I am thinking, what if we have EddieHub as community partner in various hackathons ? What do you think ? :thinking:

**Reply by naomi-lgbt** [reply]
I think it depends on the hackathon and whether they are open to community partnerships - and what kind of time commitment would be required on @eddiejaoude's part.

However, I think it's a great idea.

**Reply by carrycooldude** [reply]
Loved the Idea

**Comment by siddharthdayalwal** [comment]
Are you a blockchain developer or have worked before in this space? I have a question ...

What's the best thing you have experienced so far and why is blockchain space worth getting into ?

**Reply by naomi-lgbt** [reply]
This is an excellent question and thank you for sharing. 🙂

**Reply by stemount** [reply]
There are a load of people who do not understand that the blockchain is not 1:1 with cryptocurrency.

You can help people understand that "blockchain" technology is not exactly the correct application. Please share with your friends, communities and colleagues about things i.e. smart contracts and using a blockchain as storage, audit and all sorts of other applications!

**Reply by siddharthdayalwal** [reply]
Agree, thanks for sharing! :+1:

**Reply by carrycooldude** [reply]
Also, DApps is also in fashion, and BLockCHain, Quantum Computing, Deep Learning, and many more are coming with some common applications like to do the stuff on Data by NASA, ISRO, and much more organization.

**Reply by ujjwal-kr** [reply]
https://builtin.com/blockchain/blockchain-applications
Found something related to your question ^

You can get started with :
start with https://docs.soliditylang.org/
https://web3js.readthedocs.io/en/v1.3.4/
https://www.freecodecamp.org/news/build-a-blockchain-in-golang-from-scratch/

**Comment by stemount** [comment]
Following on from the thread about getting stuck with ES6:

ES6 and TypeScript is a slow burner with learning; always document your learning (LEARNING.md) and I would always recommend practicing the habit of writing issues. You will find that TypeScript concepts and paradigms will be similar to Java, Swift, Python, PHP 8, C#... you get the point 😉  - so if you have experience with other languages from formal education or any other projects - find similarities between languages.

e.g. if you were on a commercial project and you were to write a bug up for you to collaborate with your colleagues, you would put as much information in there so your colleagues can help quicker, especially when the world will be all working remotely!

So, I would say that writing skills are even more important than technical merit/skill!

**Reply by naomi-lgbt** [reply]
This is a great idea.

I'd encourage folks to create a repository to keep their notes and learning journeys in.

**Comment by khalidk799** [comment]
Any Suggested Platforms for Learning React? I've been Learning from freecode camp & some random youtube explanation when I am stuck.

**Reply by naomi-lgbt** [reply]
freeCodeCamp was where I learned - then I started building projects and contributing to open source. In my opinion, the best way to learn is by building stuff.

**Comment by starlightknown** [comment]
A question for all the community members, leaders- share your journey which could help someone who are completely new , how did you start in opensource or hackathons, how did you start a community or lead a community and collaborate? What are good practices? About work or internships

**Reply by naomi-lgbt** [reply]
For me, it's all about focusing on helping people.

I don't join a community with the goal of becoming a leader. I join the community with a goal of helping people learn, grow, and become more. As you do that, you'll find people gravitate toward you because of your genuine desire to support others.

**Reply by eddiejaoude** [reply]
I think people should make a step, they might go forward or left or right, but all of it is forward - they will only know which direction after they take that step.  I suggestion people keep try to keep moving little and often (sounds like open source too 🤔 😂 )

**Reply by naomi-lgbt** [reply]
> I think people should make a step,

This is huge. You cannot get anywhere if you do not take the first step.

I think folks should not be afraid to make mistakes - I've certainly made more than my share of them. But failures are only the stepping stones on the path to success, and so long as you keep moving you *will get there*.

**Comment by carrycooldude** [comment]
What're your views on Designathon ??
and UI/UX is a crucial part of any product but people took design taken for granted.

**Reply by naomi-lgbt** [reply]
This is my first time hearing of Designathon. Would you share some links and resources for folks to learn more about it?

**Reply by Rahma-AlWadhahi** [reply]
Hmm, does it include applying design sprint?

**Reply by carrycooldude** [reply]
It's Just a hackathon  for designers and UI/UX Enthusiast for building any product or idea

**Reply by stemount** [reply]
Absolutely! You can find that if you stop and take a second when building a product; ascertain what it's supposed to do first and you may find that 80% of the features you think are required do not even meet the project goal (or even the sprint goal).

Sharing technologies like Figma, Adobe xD, Sketch including all of your resources and UI kits would allow people to appreciate the design prototyping stage.

**Comment by adityaraute** [comment]
Pursuing a Data Science specialization through Coursera and I am on the brink of completing the Course 2 / 5 which is MatPlotLib.

(Sorry, cannot speak.)

**Reply by naomi-lgbt** [reply]
That is great - keep up the learning and happy coding.

Do you have any resources you can share that relate to your learning?

**Reply by eddiejaoude** [reply]
MatPlotLib? Matlab?

**Reply by Rahma-AlWadhahi** [reply]
It’s a Python library

**Reply by adityaraute** [reply]
I mean I could link:
- The [Course](https://www.coursera.org/specializations/data-science-python) available on Coursera
- Some [principles](https://medium.com/nightingale/improve-your-visualization-skills-using-tuftes-principles-of-graphical-design-3a0f40a53a2c) for data visualization.
- Some interesting [findings](http://www.tylervigen.com/spurious-correlations)

But I feel kinda lazy right now

**Comment by Rahma-AlWadhahi** [comment]
Quick question: does anyone have problems focusing on the audio discussion and read the threads here like me? 😂

**Reply by carrycooldude** [reply]
Yes Me

**Reply by eddiejaoude** [reply]
yes 👍 - I thought it was just me, thank you for sharing ❤️

**Reply by naomi-lgbt** [reply]
I have been told I am part robot, and can juggle both at once :P

**Reply by stemount** [reply]
It took me 50 mins to understand what thread we were following :-p

**Comment by stemount** [comment]
**Things I have done/learned this week:**
- [x] Absolutely nothing about programming.
- [x] Got into the habit of preparing all my meals after 5-6 years of total laziness (dining out, takeaways!!!!!)

**Things that have shocked me:**
- [ ] How a 21 year old me definitely took some very different takeaways from the books I've re-read on the right
- [ ] How important nutrition is for the brain and even improves sleep (selenium-rich last meal of the day)
- [ ] How cooking for other people is just a magical experience.

Books I have picked up and are in the queue - documenting my learning as a JPEG :-D

![IMG_0274](https://user-images.githubusercontent.com/150512/109397061-b9f07e80-792c-11eb-8933-578368d5fdfb.jpeg)

**Reply by khalidk799** [reply]
Books 🔥

**Reply by adityaraute** [reply]
No fiction?

**Reply by eddiejaoude** [reply]
Anne Boden 💥  StarlingBank 👍

**Reply by Nitesh-thapliyal** [reply]
I love reading books there are some books on my wishlist :
- Atomic Habit by James Clear
- Deep Work by Cal Newport
- Can't Hurt Me by David Goggins
- Why We Sleep by Matthew walker

**Reply by stemount** [reply]
@eddiejaoude I actually use Blinkist (app) which is a book summarised in 13 minutes to actually rate a book before I read it, a bit like watching the film before you get the book because I always feel let down when I've made a bad book choice or I'm labouring/forcing myself to read!

**Reply by stemount** [reply]
> Deep Work by Cal Newport

I have read this and I really enjoyed the ideas; it reminds me of being a politician e.g. you have loads of input points (social media, letters, press).

**Comment by carrycooldude** [comment]
Does anyone have any views on VR, AR, XR on upcoming techs???

**Reply by khalidk799** [reply]
As of I think, VR has a very bright future in Game Dev ofcourse, The release of Movies like Ready Player one have created a great demand in Gamers. And Also VRs are cool and exciting so why not. ( Can't speak rn p)

**Reply by stemount** [reply]
@carrycooldude I think VR is really far off being good because of the hardware requirements still (clunky headset, USB-C cables are short, you need specific gaming PCs with the correct graphics card sometimes), however if something like Second Life happens again e.g. Steam VR chat in a way that isn't "edgy" and can be for everybody e.g. you could have a professional career in "Pro VR Chat Life" that would be interesting to meet new people.

AR is what Apple have double downed on recently which I think is the way forward because it does not require any headset, google cardboard or even specialist phones, iPhone 8+ supports AR.

**Comment by oreo-gif** [comment]
apart from FreeCodeCamp and some mini projects should I be doing anything else for learning in this field?

**Reply by khalidk799** [reply]
Make your own portfolio website

**Reply by Rahma-AlWadhahi** [reply]
I think that sharing your knowledge with other people or keep iterating and improving these mini projects would be good

**Reply by eddiejaoude** [reply]
I think joining a hackathon is really useful 🤓 . EddieHub is joining a hackathon again soon, you should join 👍 EddieHub won last time, everyone here was awesome 💥

**Reply by oreo-gif** [reply]
I have already started pulling my friends towards knowing about open source :P @Rahma-AlWadhahi

**Reply by eddiejaoude** [reply]
> I have already started pulling my friends towards knowing about open source

YAY 🎉

**Reply by oreo-gif** [reply]
Got it @eddiejaoude :)

**Reply by Nitesh-thapliyal** [reply]
yes hackathons are the best way to learn and grow,  Hackathons teaches us how to collaborate and work together💪

**Comment by stemount** [comment]
I'm going to cheat and add something to this thread... make sure you connect GitHub/YouTube/whatever to Discord so we can find your profiles and give you some Github Love by following you or starring projects or whatever :-) 

for example @francescoxx has their profile details connected in Discord, some asked how to do this (it's not really obvious what a "connected account" **is**) and it's in `User Settings > Connections` on Discord. It's hard to keep up with names vs GitHub names vs Discord nicknames!

**Reply by eddiejaoude** [reply]
Thanks @stemount 👍
